doc/classes/TextEdit.xml

```diff
@@ -235,7 +243,7 @@
 			<argument index="0" name="line" type="int">
 			</argument>
 			<description>
-				Returns if the given line is folded.
+				Returns if the line at the specified index is folded or not.
```

Comment:
"Returns whether" would be better.

Reply:
BTW, I wonder if it's better to use "or not" in "Returns whether ... or not" or if it should be omitted.
CC @cbscribe @NathanLovato. This should likely go in the docs writing guidelines too as "Returns whether ..." should likely be the recommended construct for any boolean getter.

Reply:
That could also be used for setters not just getters. Example:
`<method name="set_line_as_hidden">`
"If [code]true[/code], hides the line of the specified index." 
Could be:
"Determines whether the line at the specified index will be hidden or not."